### ISSUE TITLE
Disabled Redirect

### ISSUE THREAD

**Issue by maurosoft1973** [matteocontrini/PlainHttp#8]
Hi Matteo,

it's possible to disabled automatic redirect?

Thanks,
Mauro

**Comment by matteocontrini** [comment]
In .NET redirects following is an option on the HTTP handler (and it defaults to true), so this would be a global option.

Would this be a solution?

It could work like this:

```csharp
string url = "https://bit.ly/41BbJeW";

HttpRequest.HttpClientFactory = new HttpClientFactory(new HttpClientFactory.HttpHandlerOptions
{
    AllowAutoRedirect = false,
});

IHttpRequest request = new HttpRequest(url);

var response = await request.SendAsync();

Console.WriteLine((int)response.StatusCode + " " + response.StatusCode);
Console.WriteLine(response.GetSingleHeader("Location"));
```

Output:

```
301 Moved
https://github.com/matteocontrini/PlainHttp
```

**Comment by maurosoft1973** [comment]
Hi, 

I sent you a PR

**Comment by matteocontrini** [comment]
Thanks. I already had a local implemention together with docs. I was asking if this is fine, I assume yes.

The feature is included in the release v2.1.0, which will be live on NuGet soon.
